src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidator.java

```diff
@@ -99,10 +99,9 @@ private void handleValidationResponse(
         switch (res.status) {
             case ERRORS:
                 throw new OcrValidationException(
-                    "Ocr validation failed. "
-                        + "Document OCR: " + docWithOcr.documentControlNumber + ". "
-                        + "Envelope: " + envelope.zipFileName + ". "
-                        + "Errors: " + Strings.join(res.errors, ", ")
+                    "OCR validation service returned OCR-specific errors. "
+                        + "Document control number: " + docWithOcr.documentControlNumber + ". "
+                        + "Envelope: " + envelope.zipFileName + ". ");
```

Comment:
We log this exception and also include it in event data.

The only concern that I have is that now the document provider will not know what exactly is wrong with OCR data. They will be able to check that with the service, though. Thoughts?